### DISCUSSION TITLE
graphql/codegen/docs/guides/graphql-server-apollo-yoga

### DISCUSSION THREAD

**Discussion by giscus** [dotansimha/graphql-code-generator#8560] [Docs Discussions]
# graphql/codegen/docs/guides/graphql-server-apollo-yoga

GraphQL Code Generator Documentation

https://www.the-guild.dev/graphql/codegen/docs/guides/graphql-server-apollo-yoga

**Comment by rohit267** [comment]
3. Configure the plugin

Create or update your codegen.yaml file as follows:

But the file shown in ts.

**Reply by charlypoly** [reply]
Hi @rohit267,

Thank you for catching this! 🦅 
It is now fixed!
